src/buzz/buzzparser.c

```diff
@@ -615,6 +617,8 @@ int parse_if(buzzparser_t par) {
       chunk_append("\tjump " LABELREF "%u", lab2);
       /* Mark this place as label 1 and keep parsing */
       chunk_append(LABELREF "%u", lab1);
+      // Allow the use of non-cuddled brackets
```

Comment:
Please avoid the C++ comment style. Use `/* comment */` instead of `// comment`.